public_suffix_list.dat

```diff
@@ -11217,6 +11217,24 @@ fbxos.fr
 freebox-os.fr
 freeboxos.fr
 
+// Futureweb OG : http://www.futureweb.at
+// Submitted by Andreas Schnederle-Wagner <schnederle@futureweb.at>
+ex.ortsinfo.at
+at.ex.ortsinfo.at
+com.ex.ortsinfo.at
+de.ex.ortsinfo.at
+eu.ex.ortsinfo.at
+net.ex.ortsinfo.at
+tirol.ex.ortsinfo.at
+futuremailing.at
```

Comment:
The sorting here is incorrect, per https://github.com/publicsuffix/list/wiki/Guidelines